src/frontend/src/components/logger.js

```diff
@@ -0,0 +1,74 @@
+/**
+	move (with character)
+	character death
+	game end
+	getting items(dracula and player both can get one)
+	exchanging items
+**/
+
+import { time } from "three/webgpu";
+
+export const LogEventTypes = {
+  MOVE: "move",
+  CHARACTER_DEATH: "character_death",
+  GAME_END: "game_end",
+  ITEM_ACQUISITION: "item_acquisition",
+  ITEM_EXCHANGE: "item_exchange",
+};
+
+export function LogInfo({
```

Comment:
Passing object because without object it makes the usuage of the function have bunch of nulls as there arguments because we dont necessarily require that many fields for a particular event.
Is there a better way to do it?